utility/src/main/java/org/oppia/android/util/parser/html/HtmlParser.kt

```diff
@@ -94,6 +94,12 @@ class HtmlParser private constructor(
       htmlContent = modifiedHtmlContent
     } else {
       htmlContentTextView.textDirection = View.TEXT_DIRECTION_LTR
+      val regex = Regex("""<oppia-noninteractive-image [^>]*>.*?</oppia-noninteractive-image>""")
```

Comment:
Per https://github.com/oppia/oppia-android/issues/4859#issuecomment-1404548538, this issue is specific to images that appear within 'li' tags, but the solution does not hint at matching regex for images in `li` tags specifically, jus images in general.

Could you kindly expound on your intended solution, and also sample other images to check that they are not wrongly centered in unexpected places?

Reply:
Actually that regex is targeting both inline and block images, here I'm wrapping image with `div` tag so that image display in block mode. This will display inline images in block mode and this regex code doesn't make any changes in images which are already being displayed in block mode.

Reply:
Attached image in description.